### PR TITLE
[ALLUXIO-2135]mark TtlIntervalRule final

### PR DESCRIPTION
**Pull request by libo1999425**
https://alluxio.atlassian.net/browse/ALLUXIO-2135

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

File: core/server/src/test/java/alluxio/master/file/meta/TtlIntervalRule.java [hunk 1]
```diff
@@ -19,13 +19,13 @@
 /**
  * Rule for temporarily setting the TTL (time to live) checking interval.
  */
-public class TtlIntervalRule implements TestRule {
+public final  class TtlIntervalRule implements TestRule {
```
Comment (jsimsa) [inline core/server/src/test/java/alluxio/master/file/meta/TtlIntervalRule.java]: remove extra space between `final` and `class`

File: core/server/src/test/java/alluxio/master/file/meta/TtlIntervalRule.java [hunk 2]
```diff
@@ -19,13 +19,13 @@
...
   private final long mIntervalMs;
 
   /**
    * @param intervalMs the global checking interval (in ms) to temporarily set
    */
-  public TtlIntervalRule(long intervalMs) {
+  public  TtlIntervalRule(long intervalMs) {
```
Comment (jsimsa) [inline core/server/src/test/java/alluxio/master/file/meta/TtlIntervalRule.java]: remove extra after `public`

**Pull request merged**